net/testdata/NamespaceTest.cs

```diff
@@ -0,0 +1,13 @@
+namespace System {
+    public class Foo {
+        public void foo(Int32 a){
+            a.
+            Int32 e;
+            e.
+        }
+
+        public void foo(Foo a){
+            a.
```

Comment:
Might be because it doesn't find Foo as a type (IIRC types need to be in a compiled dll/exe).